test/gov/agbot_del_loop.sh

```diff
@@ -52,7 +52,7 @@ do
       sleep 180
 
       echo -e "Current workload usages\n"
-      curl -sS http://localhost:81/workloadusage | jq -r '.'
+      curl -sS http://localhost:8082/workloadusage | jq -r '.'
```

Comment:
Should it be 8511?

Reply:
No, 8082 is correct. This is the port for the agbot. It's the same port which is used earlier in the script.